Goals failing with Access Configuration

<!-- Please fill out the following information to help us in analyzing the issue, but feel free to skip it if you don't have the information at hand or if it does not apply. Please remove every section which does not apply to the issue. -->
### Description
After I defined Access Configuration for k8s cluster all plugin goals fail with same error message:
```
[ERROR] Failed to execute goal io.fabric8:fabric8-maven-plugin:4.0.0:build (default-cli) on project <project name>: Unable to parse configuration of mojo io.fabric8:fabric8-maven-plugin:4.0.0:build for parameter access: Cannot create instance of class io.fabric8.maven.core.access.ClusterConfiguration: Class org.codehaus.plexus.component.configurator.converters.AbstractConfigurationConverter can not access a member of class io.fabric8.maven.core.access.ClusterConfiguration with modifiers "private" -> [Help 1]
```

### Info

* f-m-p version : 4.0.0
* Maven version (`mvn -v`) : 
```
Apache Maven 3.5.2
Maven home: /usr/share/maven
Java version: 1.8.0_201, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-46-generic", arch: "amd64", family: "unix"
```
* Kubernetes / OpenShift setup and version : 1.8.5
* If it's a bug, how to reproduce :
Define any Access Configuration parameter with Property (maven or system); or define any Access Configuration parameter in plugin configuration.  `<access>` block can be even empty.    

Sample minimal pom.xml that is failing:
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <groupId>com.example</groupId>
    <artifactId>fmp</artifactId>
    <version>1.0.0</version>
    <build>
        <plugins>
            <plugin>
                <groupId>io.fabric8</groupId>
                <artifactId>fabric8-maven-plugin</artifactId>
                <version>4.0.0</version>
                <configuration>
                    <access>
                        <masterUrl>https://example.com</masterUrl>
                    </access>
                </configuration>
            </plugin>
        </plugins>
    </build>
</project>
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,6 +25,7 @@ After this we will switch probably to real [Semantic Versioning 2.0.0](http://se
 * Refactor 1592: Removed Deprecated/Unused Osio specific enrichers.
 * Fix 1608: Added Enricher for namespace to project conversion.
 * Feature 1611: Allow creating namespaces from xml configuration
+* Fix 1616: Goals failing with Access Configuration
 
 ### 4.0.0 (14-03-2019)
 * Refactor 678: Enricher Workflow to allow direct generation of OpenShift objects.

--- a/core/src/main/java/io/fabric8/maven/core/access/ClusterConfiguration.java
+++ b/core/src/main/java/io/fabric8/maven/core/access/ClusterConfiguration.java
@@ -42,7 +42,7 @@ public class ClusterConfiguration {
     private String keyStoreFile;
     private String keyStorePassphrase;
 
-    private ClusterConfiguration() {
+    public ClusterConfiguration() {
         this.namespace = KubernetesHelper.getDefaultNamespace();
     }
 

